pages/tutorials/graphql_sso.md.erb

```diff
@@ -0,0 +1,250 @@
+# Manual SSO Setup with GraphQL
+
+Buildkite’s SSO login can be set up by emailing support, or you can set it up manually using our [GraphQL APIs](/docs/apis/graphql). This tutorial covers how to set up SSO manually using GraphQL, for most of the common SSO providers.
+
+For details on every available option available in the GraphQL APIs, please use the documentation sidebar built into the [GraphQL Explorer](/docs/graphql-api#getting-started).
+
+<%= toc %>
+
+## Finding your organization’s GraphQL ID
+
+For every type of SSO provider, you’ll need your Buildkite organization’s GraphQL ID. You can find the organization `id` using the following GraphQL query:
+
+```graphql
+query OrgId {
+  organization(slug: "myorg") {
+    id
+  }
+}
+```
+
+## Setting up G Suite
+
+The first step in setting up a G Suite SSO provider is to use the `ssoProviderCreate` mutation.
```

Comment:
Could we bring back the line here that explains what the example code will do? I guess the audience of this doc is going to be familiar with graphql... but I'd still think that explaining examples reduces the mental overhead for everyone.

Reply:
Cool yep, I'll fix that now!